packages/forma-36-react-components/.size-limit

```diff
@@ -3,7 +3,7 @@
       "path": "dist/index.js",
       "name": "gzipped js",
       "gzip": false,
-      "limit": "282 KB",
+      "limit": "289 KB",
```

Comment:
I guess this increase is caused by `is-hotkey`

Reply:
@arpl How do you feel about stripping it out? This is the only component that uses it and it should be pretty simple to replace, but I also see the convenience of having it if we start using it elsewhere.

Reply:
@m10l Scope of this PR is to port the component from user-interface. I agree it's beneficial to keep the bundle small let me know if this is a hard dependency for a merge.

Reply:
@arpl Ah okay, no worries then - something we can do after merging